redbeams/src/main/java/com/sequenceiq/redbeams/converter/RedbeamsConverterUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.redbeams.converter;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import javax.inject.Inject;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RedbeamsConverterUtil {
```

Comment:
This class is identical to `ConverterUtil` in core-api. Unless `ConverterUtil` is going away soon, this one is redundant and should be removed, and `RedbeamsApplication` can include `com.sequenceiq.cloudbreak.api.util` as another package for component scan.

Reply:
Deleted.